frontend/gatsby-config.js

```diff
@@ -24,6 +24,7 @@ module.exports = {
     {
       options: {
         display: `minimal-ui`, // icon: `src/images/gatsby-icon.png`, // This path is relative to the root of the site.
+	crossOrigin: `use-credentials`,
```

Comment:
Send cookie headers to the manifest endpoint (this enables serving the manifest via the OAuth proxy without getting an auth redirect)